lib/kafo/kafo_configure.rb

```diff
@@ -316,6 +316,25 @@ def app_option(*args, &block)
       self.class.app_option(*args, &block)
     end
 
+    def terminal_log_levels_message
+      if ARGV.include?('--full-help')
```

Comment:
I think you can use `kafo.full_help?` here instead, which is cleaner.

Reply:
aaaaaaaaah, yeah, THAT is possible. you're still initing clamp here, so it didn't parse stuff. you're right.